pkgs/applications/misc/easystroke/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub, fetchpatch, pkgconfig, boost, dbus_glib, gnome3,
+ xorg, help2man, intltool, libsigcxx, gcc7, wrapGAppsHook}: 
+
+with stdenv.lib;
+
+let
+  version = "0.6.0-9-gf7c1614";
+in stdenv.mkDerivation {
+  name = "easystroke-${version}";
```

Comment:
According to our naming guide (https://nixos.org/nixpkgs/manual/#sec-package-naming) the package name should probably be (final form after evaluation): `easystroke-unstable-2000-01-31` (with correct date)